dashboard/client/src/pages/job/hook/useJobProgress.ts

```diff
@@ -15,21 +15,42 @@ import {
 } from "../../../type/job";
 import { TypeTaskStatus } from "../../../type/task";
 
-const TASK_STATE_NAME_TO_PROGRESS_KEY: Record<
-  TypeTaskStatus,
+export enum TaskStatus {
```

Comment:
The useJobProgress function is related to this pull request (https://github.com/ray-project/ray/pull/35294). I will wait for the PR to be merged